src/staticModels/EditorModel.ts

```diff
@@ -19,8 +19,8 @@ export class EditorModel {
 
     public static viewPortHelper: ViewPortHelper;
 
-    public static isLoading: boolean = false;
```

Comment:
Why did we remove default values here?

Reply:
the boolean default is always false. that's might be removed by the global linter fix.

Reply:
> ![image](https://user-images.githubusercontent.com/28984914/180562307-d8b3f3eb-67b5-4ea8-958f-2939fa3ba468.png) the TextField is fixed. solving another bug.

Please remove the outline. I assume `variant="standard"` can do it for us.

Reply:
> the boolean default is always false. that's might be removed by the global linter fix.

Please let's keep these explicit false values.